### DISCUSSION TITLE
deploy multi branch on vercel

### DISCUSSION THREAD

**Discussion by m0x61h0x64i** [vercel/vercel#8576] [Help] [answered]
how can i deploy multi branch on vercel? like dev and main.

**Comment by amyegan** [comment, marked as answer]
Hi @m0x31h0x64i! Can you explain a little more about what you want to do?

Based on my assumption, these are some more resources that might help:
- [Vercel for GitHub](https://vercel.com/docs/concepts/git/vercel-for-github#) will automatically deploy your production branch to your Project URL and other branches to preview URLs
- You can also set up a staging environment to deploy from a specific branch to a staging address with a custom domain. The guide for that is called [How do I set up a staging environment on Vercel?](https://vercel.com/guides/set-up-a-staging-environment-on-vercel)
